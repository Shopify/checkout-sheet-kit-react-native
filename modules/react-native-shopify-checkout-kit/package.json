{
  "name": "react-native-shopify-checkout-kit",
  "license": "MIT",
  "version": "0.1.0",
  "main": "lib/commonjs/index.js",
  "types": "src/index.tsx",
  "source": "src/index.tsx",
  "module": "lib/module/index.js",
  "description": "A React Native library for Shopify's Checkout Kit.",
  "author": "Shopify",
  "homepage": "https://github.com/shopify/checkout-kit-react-native",
  "publishConfig": {
    "registry": "https://registry.npmjs.org/"
  },
  "scripts": {
    "clean": "rm -rf lib",
    "build": "bob build",
    "lint": "yarn typecheck && eslint src",
    "prepack": "yarn clean && yarn build",
    "typecheck": "tsc --noEmit"
  },
  "files": [
    "LICENSE",
    "README.md",
    "package.json",
    "src",
    "ios",
    "android",
    "lib",
    "*.podspec",
    "!ios/build",
    "!android/build",
    "!android/gradle",
    "!android/gradlew",
    "!android/gradlew.bat",
    "!android/local.properties",
    "!**/*.spec.*",
    "!**/*.test.*",
    "!**/.*"
  ],
  "keywords": [
    "react-native",
    "shopify",
    "checkout"
  ],
  "peerDependencies": {
    "react": "*",
    "react-native": "*"
  },
  "devDependencies": {
    "react-native-builder-bob": "^0.23.1",
    "typescript": "^5.3.2"
  },
  "react-native-builder-bob": {
    "source": "src",
    "output": "lib",
    "targets": [
      "commonjs",
      "module",
      [
        "typescript",
        {
          "project": "tsconfig.build.json"
        }
      ]
    ]
  }
}
