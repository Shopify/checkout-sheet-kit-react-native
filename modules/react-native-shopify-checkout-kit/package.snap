[
  "LICENSE",
  "README.md",
  "android/build.gradle",
  "android/gradle.properties",
  "android/src/main/AndroidManifest.xml",
  "android/src/main/AndroidManifestNew.xml",
  "android/src/main/java/com/shopifycheckoutkit/CustomCheckoutEventProcessor.java",
  "android/src/main/java/com/shopifycheckoutkit/ShopifyCheckoutModule.java",
  "android/src/main/java/com/shopifycheckoutkit/ShopifyCheckoutPackage.java",
  "ios/ShopifyCheckout-Bridging-Header.h",
  "ios/ShopifyCheckout.mm",
  "ios/ShopifyCheckout.swift",
  "lib/commonjs/index.d.js",
  "lib/commonjs/index.d.js.map",
  "lib/commonjs/index.js",
  "lib/commonjs/index.js.map",
  "lib/module/index.d.js",
  "lib/module/index.d.js.map",
  "lib/module/index.js",
  "lib/module/index.js.map",
  "lib/typescript/src/index.d.ts",
  "lib/typescript/src/index.d.ts.map",
  "package.json",
  "react-native-shopify-checkout-kit.podspec",
  "src/index.d.ts",
  "src/index.tsx"
]
